components/Header.js

```diff
@@ -9,7 +9,7 @@ const Container = styled.header`
   z-index: 9;
   color: #fff;
   background: ${(props) => props.theme.gradient.primary};
-  padding: 15px;
+  padding: 12px 0;
```

Comment:
You should Specify the height because if the image load fail, it will have different dimensions